### ISSUE TITLE
`BaseSettings` has been moved to the `pydantic-settings` package.

### ISSUE THREAD

**Issue by deliaBlue** [initialcommit-com/git-dummy#1]
Hey! I  just installed `git-dummy` and when trying to display the option the following error appeared:

```
Traceback (most recent call last):
  File "/home/****/mambaforge/envs/bioconda/bin/git-dummy", line 5, in <module>
    from git_dummy.__main__ import app
  File "/home/****/mambaforge/envs/bioconda/lib/python3.8/site-packages/git_dummy/__main__.py", line 8, in <module>
    from git_dummy.settings import settings
  File "/home/****/mambaforge/envs/bioconda/lib/python3.8/site-packages/git_dummy/settings.py", line 2, in <module>
    from pydantic import BaseSettings
  File "/home/****/mambaforge/envs/bioconda/lib/python3.8/site-packages/pydantic/__init__.py", line 206, in __getattr__
    return _getattr_migration(attr_name)
  File "/home/****/mambaforge/envs/bioconda/lib/python3.8/site-packages/pydantic/_migration.py", line 279, in wrapper
    raise PydanticImportError(
pydantic.errors.PydanticImportError: `BaseSettings` has been moved to the `pydantic-settings` package. See https://docs.pydantic.dev/2.0/migration/#basesettings-has-moved-to-pydantic-settings for more details.

For further information visit https://errors.pydantic.dev/2.0/u/import-error
```

For the installation I used

```
pip3 install git-dummy
```

The `git-dummy` version is `0.0.8` with a `python` version is `3.8.17` running in `Ubuntu 22.04.2`.

During the installation, these other dependencies were installed:

```
annotated-types-0.5.0 
pydantic-2.0 
pydantic-core-2.0.1 
typer-0.9.0
```

Thank you!

**Comment by initialcommit-io** [comment]
> when trying to display the option

Hi @deliaBlue - can you let me know exactly what command you're running that's generating the error?

**Comment by deliaBlue** [comment]
Sure! The command that displayed the error is

```
git-dummy -h
```

**Comment by initialcommit-io** [comment]
Thanks for this - I can reproduce the issue now. It looks like the pydantic dependency got updated behind the scenes which caused issues with git-dummy and actually git-sim as well.  I'll be working on this and will let you when I release a fix.

**Comment by initialcommit-io** [comment]
@deliaBlue This should be fixed in git-dummy v0.0.9.  You can update to that version by running `pip install git-dummy --upgrade`.

Please test it out and let me know if any issues.  I will close this for now, but feel free to reopen if anything comes up.
